meteor/app/client/blockFDQN.js

```diff
@@ -0,0 +1,63 @@
+/*
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+Copyright (c) 2014 Mozilla Corporation
+ */
+
+if (Meteor.isClient) {
+
+    Template.blockIPform.rendered = function() {
```

Comment:
The format is Template.<templatename>.event so for this it should be Template.blockFQDNform.rendered since you want it to autofill a block in the fqdn form.

Reply:
fixed